instructions.json

```diff
@@ -18,25 +18,35 @@
             "defaultInstruction": "Head {modifier}[ on {way_name}]"
         },
         "end of road": {
-            "defaultInstruction": "Turn {modifier}[ onto {way_name}]"
+            "defaultInstruction": "Turn {modifier}[ onto {way_name}] at the end of the road",
```

Comment:
I personally don't like `at the end of the road` because it is really long. I'd imagine `end of road` could be used for a special turn symbol, but should not find it's way into text instructions.

/cc @1ec5 @MoKob